examples/jetstream/limits-stream/java/Main.java

```diff
@@ -0,0 +1,135 @@
+            futures.add(js.publishAsync("events.input_changed", null));
+            futures.add(js.publishAsync("events.input_blurred", null));
+            futures.add(js.publishAsync("events.key_pressed", null));
+            futures.add(js.publishAsync("events.input_focused", null));
+            futures.add(js.publishAsync("events.input_changed", null));
+            futures.add(js.publishAsync("events.input_blurred", null));
+
+            CompletableFuture<Void> publishCompleted = CompletableFuture.allOf(futures.toArray(new CompletableFuture[0]));
+            try {
+                publishCompleted.join();
+                System.out.println("Published 6 messages.");
+            } catch (Exception e) {
+                System.out.println("Publish failed " + e);
+            }
+
+            // Checking out the stream info, we can see how many messages we
+            // have.
+            printStreamState(jsm, streamName);
+
+            // Stream configuration can be dynamically changed. For example,
+            // we can set the max messages limit to 10 and it will truncate the
+            // two initial events in the stream.
+            streamConfigBuilder.maxMessages(10);
+            jsm.updateStream(streamConfigBuilder.build());
+            System.out.println("Set max messages to 10.");
+
+            // Checking out the info, we see there are now 10 messages and the
+            // first sequence and timestamp are based on the third message.
+            printStreamState(jsm, streamName);
+
+            // Limits can be combined and whichever one is reached, it will
+            // be applied to truncate the stream. For example, let's set a
+            // maximum number of bytes for the stream.
+            streamConfigBuilder.maxBytes(300);
+            jsm.updateStream(streamConfigBuilder.build());
+            System.out.println("Set max bytes to 300.");
+
+            // Inspecting the stream info we now see more messages have been
+            // truncated to ensure the size is not exceeded.
+            printStreamState(jsm, streamName);
+
+            // Finally, for the last primary limit, we can set the max age.
+            streamConfigBuilder.maxAge(Duration.ofSeconds(1));
+            jsm.updateStream(streamConfigBuilder.build());
+            System.out.println("Set max age to one second.");
+
+            // Looking at the stream info, we still see all the messages..
+            printStreamState(jsm, streamName);
+
+            // until a second passes.
+            System.out.println("Sleeping one second...");
+            Thread.sleep(1000);
+
+            printStreamState(jsm, streamName);
+        } catch (InterruptedException | IOException | JetStreamApiException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private static void printStreamState(JetStreamManagement jsm, String streamName) throws IOException, JetStreamApiException {
```

Comment:
Can you add a label parameter to this call, so you can give a little more detail on the output, like "Initial State", "After Changing Max Age and Sleeping" etc.

Reply:
Done